stories/actions/update.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from '@storybook/react';
+import { Layout } from '../utils/Layout';
+import {
+  RoiContainer,
+  RoiList,
+  RoiProvider,
+  useCommittedRois,
+  useRoiActions,
+  useRoiState,
+} from '../../src';
+import { Image } from '../utils/Image';
+import { initialRois } from './utils';
+
+export default {
+  title: 'Actions',
+} as Meta;
+
+export function Update() {
+  return (
+    <Layout>
+      <RoiProvider initialRois={initialRois}>
+        <div style={{ display: 'flex', flexDirection: 'column', gap: 5 }}>
+          <Internal />
+
+          <RoiContainer target={<Image />}>
+            <RoiList />
+          </RoiContainer>
+        </div>
+      </RoiProvider>
+    </Layout>
+  );
+}
+
+function Internal() {
```

Comment:
This comment is valid for all the places where you create an `Internal` component.
- The name is not appropriate.
- Put it inside of the story (ignore the eslint warning if there is one). That way we can see how it works in the story's source code.

Reply:
we can't do that. everytime this function exist it use the context, that i create on the main function

Reply:
i can only, rename the "Internal" function to be more specific. But i really need this to call functions inside the context

Reply:
I'm just suggesting to move where the function is defined, not where the component is rendered.

> we can't do that. everytime this function exist it use the context, that i create on the main function